bin/run_functions.js

```diff
@@ -47,24 +48,28 @@ module.exports.dispatcher = function (args) {
             printModuleHelp();
             break;
         default:
-            util.error("Unknown operatation");
+            util.error("Unknown operation");
             break;
     }
 };
 
 function printModuleHelp() {
     util.error("Usage: testengine run <command>");
     util.error("Commands: ");
-    util.error("   project [testsuite=<name>] [async] [skipdeps] [testcase=<name>] [timeout=<seconds>] [tags=(tag1,tag2)] [output=<directory>] [reportFileName=<filename>] [format=junit/excel/json/pdf] [environment=<environment name>]");
+    util.error("   project [testsuite=<name>] [async] [skipdeps] [testcase=<name>] [securitytest=<name>] [timeout=<seconds>] [tags=(tag1,tag2)] [output=<directory>] [reportFileName=<filename>] [format=junit/excel/json/pdf] [environment=<environment name>]");
     util.error("           [projectPassword=<password>] [proxyHost=<hostname>] [proxyPort=<port>] [proxyUser=<username>]");
     util.error("           [proxyPassword=<password>] <filename>");
     util.error("   help");
 }
 
 function conflictingOptionsCheck(options) {
+    if (('securitytest' in options) && (('testcase' in options) || ('testsuite' in options))) {
+        util.error('Error: You can specify only functional test (testsuite and/or testscase) or security test (securitytest) in one go');
```

Comment:
It is also a functional to test to run the entire project (by not specifying neither of the options).
In the jenkins plugin we use "Parameters testSuiteName and testCaseName are not allowed when securityTestName is used".

Reply:
good point, thank you